openmetadata-ui/src/main/resources/ui/src/pages/DataInsightPage/KPIList.tsx

```diff
@@ -122,21 +123,42 @@ const KPIList = () => {
         key: 'actions',
         render: (_, record) => {
           return (
-            <Tooltip
-              placement="left"
-              title={
-                isAdminUser ? t('label.delete') : NO_PERMISSION_FOR_ACTION
-              }>
-              <Button
-                data-testid={`delete-action-${getEntityName(record)}`}
-                disabled={!isAdminUser}
-                icon={
-                  <SVGIcons alt="delete" icon={Icons.DELETE} width="18px" />
-                }
-                type="text"
-                onClick={() => setSelectedKpi(record)}
-              />
-            </Tooltip>
+            <Space>
+              <Tooltip
+                placement="left"
+                title={
+                  isAdminUser ? t('label.edit') : NO_PERMISSION_FOR_ACTION
```

Comment:
Can we use the existing localization key here instead of `NO_PERMISSION_FOR_ACTION`?